packages/core/src/codewhisperer/region/regionProfileManager.ts

```diff
@@ -120,7 +121,18 @@ export class RegionProfileManager {
         return this._profiles
     }
 
-    constructor(private readonly authProvider: IAuthProvider) {}
+    constructor(private readonly authProvider: IAuthProvider) {
+        const getProfileFunction = () =>
+            globals.globalState.tryGet<{ [label: string]: RegionProfile }>('aws.amazonq.regionProfiles', Object, {})
```

Comment:
There are multiple instances of this tryGet in this file. Can we make a single private method and just call that